agent/stats/queue_test.go

```diff
@@ -122,26 +125,43 @@ func getPredictableHighMemoryUtilizationInBytes(size int) []uint64 {
 	return memBytes
 }
 
-func createQueue(size int, predictableHighMemoryUtilization bool) *Queue {
+func getPredictableInt64Overflow(size int) []uint64 {
```

Comment:
this method name is a bit misleading.. how about calling it getLargeInt64Stats ?

Reply:
sure.  I'll make that change.